### PR TITLE
Location service

### LINKED ISSUE #37: add ability and mock out listing filter for getListings

### PR DESCRIPTION
**Pull request by jdcarey128**
## Description

This PR creates a LocationService. This will ultimately be used to query our microservice for zip codes. This PR also enables the cors gem that should allow FE to access the db. Finally, this PR updates the getListings query to have the ability to filter listings by zip code and radius. 
closes #37 

## Motivation and Context

Process in building a new feature and bug fix. 

## How Has This Been Tested?

Full testing for the service. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] I have updated the README to reflect any changes.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -26,7 +26,7 @@ gem 'puma', '~> 3.11'
 gem 'bootsnap', '>= 1.1.0', require: false
 
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin AJAX possible
-# gem 'rack-cors'
+gem 'rack-cors'
```
Comment (dcoleman21) [inline Gemfile:29]: Is this what helps make calls from separate ports?

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@
 SimpleCov.start
 SimpleCov.add_filter ['app/controllers/graphql_controller.rb', 'app/graphql/garden_share_be_schema.rb', 'spec/', 'config/']
 
+require 'webmock/rspec'
```
Comment (dcoleman21) [inline spec/rails_helper.rb:6]: Are we using python?

**Review by dcoleman21 — APPROVED** [review]
Is this the microservice we were working on yesterday?

**Review by Callbritton — COMMENTED** [review]
Looks like we need to sort out the env variable for Travis before merging!

**Review by Callbritton — APPROVED** [review]

**Pull request merged**